portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,33 @@
   </head>
   <body>
     <div id="content">
+
+
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>This is Eddy's portfolio.</p>
+      <p>Hello! My name is Eddy Nkere and my hometown is Charlottesville Virginia.</p>
+      <p>One thing that I am quite passionate about is the story line of syfy characters in movies and TV-shows.</p>
+      <p> One of them is the Mad Titan and the Black Order</p>
+      <img src="img/thanos2.jpg" width = "500" height= "300"/>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click here to learn the names of each one of the children of Thanos:</p>
+      <button onclick="addRandomGreeting()">Click</button>
       <div id="greeting-container"></div>
-    </div>
+      <h2>Background</h2>
+      <p>I was born in the Republic Democratic of Congo, and moved to the United State in 2015.
+          I love being active, by exercising and being in conection with nature. I love board games like: chess, checker, and sudoku. They help me improve my critical and analical ways of thinking. 
+          I am passionate about the Computer Science field for the opportunities it provides to inovate and create new technologies.</p>
+          
+       <h2>Projects</h2>
+       <p>I have worked on projects such as:</p>       
+       <p>MineField, Imaginary-Soldier, and Class-Grades</p>
```

Comment:
Optional - You can create lists in HTML by using the `ul` and `ol` tags for unordered and ordered lists, respectively:

For example here you would do:

```
<ul>
  <li>MineField</li>
  <li>Imaginary-Soldier</li>
  <li>Class-Grades</li>
</ul>
```

Reply:
Yeah I didn't know this, I was thinking about this too when I was making the page.